apps/typegpu-docs/src/content/examples/rendering/cubemap-reflection/helpers.ts

```diff
@@ -0,0 +1,47 @@
+import tgpu from 'typegpu';
+import * as d from 'typegpu/data';
+import * as std from 'typegpu/std';
+
+export const unpackVec2u = tgpu['~unstable'].fn(
+  { packed: d.vec2u },
+  d.vec4f,
+)(({ packed }) => {
+  const xy = std.unpack2x16float(packed.x);
+  const zw = std.unpack2x16float(packed.y);
+  return d.vec4f(xy, zw);
+});
```

Comment:
Since we're keeping both versions, we could keep these in the "old" API 🤔. And the same for pack(...).

```suggestion
export const unpackVec2u = tgpu['~unstable'].fn(
  [d.vec2u],
  d.vec4f,
)((packed) => {
  const xy = std.unpack2x16float(packed.x);
  const zw = std.unpack2x16float(packed.y);
  return d.vec4f(xy, zw);
});
```